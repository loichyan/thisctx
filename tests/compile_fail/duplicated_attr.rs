use thisctx::WithContext;

#[derive(WithContext)]
enum Error {
    #[thisctx(no_skip)]
    #[thisctx(skip)]
    Variant1,
    #[thisctx(no_skip)]
    #[thisctx(no_skip)]
    Variant2,
}

fn main() {}
