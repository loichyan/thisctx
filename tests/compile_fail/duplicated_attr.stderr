error: duplicate #[thisctx(skip)] option
 --> tests/compile_fail/duplicated_attr.rs:6:15
  |
6 |     #[thisctx(skip)]
  |               ^^^^
